.github/workflows/main.yml

```diff
@@ -0,0 +1,59 @@
+name: Build
```

Comment:
I'd rename this file from `.github/workflows/main.yml` to `.github/workflows/build.yml`